### PR TITLE
Feature/infinite looping

### PR DESCRIPTION
**Pull request by leandrooriente**
Original:
https://github.com/Vermonster/react-responsive-carousel/commit/f80024a81bfe804958835668c6dc8e6425049a40

### CODE REVIEW

File: src/__tests__/Carousel.js [hunk 1]
```diff
@@ -139,22 +139,37 @@ describe("Slider", function() {
...
+
 			TestUtils.Simulate.click(componentInstance['item6']);
 			expect(ReactDOM.findDOMNode(componentInstance).querySelectorAll('.carousel-slider .control-next.control-disabled').length).toBe(1);
-		});		
-	})	
+		});
+
+		// describe("Infinite Loop", function () {
+		// 	beforeEach(function () {
+		// 		componentInstance.infiniteLoop = true;
```
Comment (leandrowd) [inline src/__tests__/Carousel.js]: `componentInstance.setProps({infiniteLoop: true});` should work here.

Soon I will get rid of these poor and old test in favour of brand new using mocha + chai + enzyme

Comment (leandrowd) [inline src/__tests__/Carousel.js]: I'll merge and update the test, no worries.

**Comment by leandrowd** [comment]
Thanks for your contribution.

**Comment by leandrowd** [comment]
Published to npm as 3.0.18 -> https://www.npmjs.com/package/react-responsive-carousel

**Comment by leandrooriente** [comment]
Thanks 😃 

I'll convert the code to ES6 soon

File: src/__tests__/Carousel.js (hunk 1 above)
Comment (leandrooriente) [inline src/__tests__/Carousel.js]: Nice

**Pull request merged**